src/openprocurement/api/__init__.py

```diff
@@ -61,6 +63,26 @@ def set_renderer(event):
         return True
 
 
+def _tender(request):
+    try:
+        # empty if mounted under a path in mod_wsgi, for example
+        path = decode_path_info(request.environ['PATH_INFO'] or '/')
+    except KeyError:
+        path = '/'
+    except UnicodeDecodeError as e:
+        raise URLDecodeError(e.encoding, e.object, e.start, e.end, e.reason)
+
+    tender_id = ""
```

Comment:
parts = path.split('/')
# only for urls with id and for tenders
if len(parts) < 4 or parts[3] != 'tenders':
    return None
tender_id = parts[4]